mods/ts/sequences/infantry.yaml

```diff
@@ -140,6 +140,7 @@ oxanna:
 slav:
 	Inherits: ^BasicInfantry
 	Inherits@attack: ^BasicInfantryAttack
+	icon: xxicon
```

Comment:
xxicon was always the fallback value for all actors without a specified cameo, so the idea of adding them to all civilians is legit IMO.